packages/block-editor/src/hooks/content-lock-ui.js

```diff
@@ -104,9 +104,13 @@ export const withBlockControls = createHigherOrderComponent(
 			return <BlockEdit { ...props } />;
 		}
 
+		const showStopEditingAsBlocks = isEditingAsBlocks && ! isContentLocked;
+		const showStartEditingAsBlocks =
+			! isEditingAsBlocks && isContentLocked && props.isSelected;
```

Comment:
I'm curious why those were needed as separate variables. Is the new version breaking these expressions somehow?

Reply:
Agreed, thank you for clarifying 👍